### ISSUE TITLE
ArgumentListWrapping reports false positive since 0.41.0

### ISSUE THREAD

**Issue by WonderCsabo** [pinterest/ktlint#1159]
I have the following code snippet:

```kotlin
class MyClass {

    private fun initCoilOkHttp() {
        Coil.setImageLoader(
            ImageLoader.Builder(this)
                .crossfade(true)
                .okHttpClient(
                    okHttpClient.newBuilder()
                        .cache(CoilUtils.createDefaultCache(this))
                        .build()
                )
                .build()
        )
    }
}
```

## Expected Behavior
Ktlint should not report any issues

## Observed Behavior
Since Ktlint 0.41.0, the `ArgumentListWrappingRule` reports the following:
```
Unexpected indentation (expected 16, actual 20)
Unexpected indentation (expected 12, actual 16)
```

It means it tells this would be correct:

```kotlin
class MyClass {

    private fun initCoilOkHttp() {
        Coil.setImageLoader(
            ImageLoader.Builder(this)
                .crossfade(true)
                .okHttpClient(
                okHttpClient.newBuilder()
                    .cache(CoilUtils.createDefaultCache(this))
                    .build()
            )
                .build()
        )
    }
}
```

## Steps to Reproduce
Run `ktlint codesnippet.kt --android --experimental`

## Your Environment

* Version of ktlint used: 0.41.0
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): command line
* Operating System and version: macOS 11.2.3

**Comment by paul-dingemans** [comment]
I have solved issue's like this in PR https://github.com/pinterest/ktlint/pull/1052 which is still waiting to be merged.

**Comment by WonderCsabo** [comment]
Thanks! However i've just built your branch and my case above still produced the false positives.

**Comment by paul-dingemans** [comment]
Sorry for miscommunication. The PR mentioned above is the first PR of a range of 5 PR's. Originally I submitted one big PR resolving many identation issues. On request of maintainers I have split it into multiple smaller PR's. The other PR's can only be submitted once the first is merged.

**Comment by shashachu** [comment]
Looks like it was introduced in this commit: https://github.com/pinterest/ktlint/commit/3591e61cd08bf7f55e342c533eb488facb231579

**Comment by osipxd** [comment]
Messed with this issue again on 0.42.1
```kotlin
fun onSendMessageClicked() {
    flow { chat.sendMessage(state.messageInput) }
        .onEach { state ->
            updateState(
                messageIsSending = state.isLoading, // <-- ArgumentListWrapping
                messageInput = "", // <-- ArgumentListWrapping
            ) // <-- ArgumentListWrapping
        }
        .launchIn(coroutineScope)
}
```

**Comment by paul-dingemans** [comment]
@osipxd You commented a closed issue. Also it is not clear what the problem or your expectation is. Maybe it is better to open a new issue?

**Comment by osipxd** [comment]
Sorry, it was due to old ktlint version
